gulpfile.babel.js

```diff
@@ -39,14 +43,20 @@ gulp.task('babel', () =>
     .pipe(gulp.dest('dist'))
 );
 
+gulp.task('build:frontend', () =>
```

Comment:
Hm, not sure. AFAIK we need Gulp to transpile the backend JavaScript so that Node knows how to serve it. Which is why I set it up to run this way. If Webpack can also handle my backend JS then that'd be great :joy: